Tools/px_romfs_pruner.py

```diff
@@ -77,7 +77,7 @@ def main():
 
                         # overwrite old scratch file
                         with open(file_path, "wb") as f:
-                                f.write(pruned_content)
+                                f.write(pruned_content.encode("UTF-8"))
```

Comment:
I'm not so sure about UTF-8. NuttX can't read that. Could you please choose a plain ASCII encoding like latin?